pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -269,6 +271,34 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			err := manageSubscriptions(ctx, namespaceList, h)
 			Expect(err).NotTo(HaveOccurred())
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// dedicated-admin user can patch consoles.operator.openshift.io CR
+		util.GinkgoIt("ded-admin user can patch consoles.operator.openshift.io CR", func(ctx context.Context) {
```

Comment:
```suggestion
		util.GinkgoIt("dedicated-admin user can patch consoles.operator.openshift.io CR", func(ctx context.Context) {
```

this will be part of the test case name

Reply:
this looks a bit weird, but it is the same for all the other test cases in this file...

Reply:
Yes, it does look a bit weird and to be honest every test case name in this file is redundant it should be fixed eventually. The full test case name for this test for example is `[Suite: informing] [OSD] dedicated-admin permissions dedicated-admin group permissions dedicated-admin user can patch consoles.operator.openshift.io CR` https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md#some-concrete-tools-at-your-disposal

```
     In particular be aware that every test is uniquely identified and described (e.g. in test reports) by the concatenation of its Describe clause and nested It clauses. So for example Describe("Pods",...).... It(""should be scheduled with cpu and memory limits") produces a sane test identifier and descriptor Pods should be scheduled with cpu and memory limits, which makes it clear what's being tested, and hence what's not working if it fails. Other good examples include:

   CAdvisor should be healthy on every node

and

   Daemon set should run and stop complex daemon

On the contrary (these are real examples), the following are less good test descriptors:

   KubeProxy should test kube-proxy

and

Nodes [Disruptive] Network when a node becomes unreachable
[replication controller] recreates pods scheduled on the
unreachable node AND allows scheduling of pods on a node after
it rejoins the cluster

An improvement might be

Unreachable nodes are evacuated and then repopulated upon rejoining [Disruptive]
```